src/commcare_cloud/ansible/deploy_citusdb.yml

```diff
@@ -16,9 +16,10 @@
     - {role: citusdb, tags: 'citusdb'}
 
 - name: pgbouncer
-  hosts: citusdb_master
+  hosts: citusdb:pg_standby
   become: true
   roles:
+    - {role: citusdb, tags: 'citusdb'}
```

Comment:
Trying to access `postgresql_data_dir` from this role. what other ways to do it ? This way it repeat what it had done above.

Reply:
You could try use `vars_files` like this: https://github.com/dimagi/commcare-cloud/blob/c14a4dd4704ba7f30c4cae6bf6411b2c319a3a5f/src/commcare_cloud/ansible/setup_pg_standby.yml#L10-L11